### PR TITLE
Added guidance for diataxis templates & xrefs

### PR DESCRIPTION
**Pull request by michelle-purcell**

### CODE REVIEW

**Review by ebullient — APPROVED** [review]

File: docs/src/main/asciidoc/doc-reference.adoc [hunk 1]
```diff
@@ -177,14 +187,14 @@ Try to write the abstract by using the following guidelines:
...
-* *Brief:* Is no more than three sentences long. 
+* *Brief:* Is no more than three sentences long.
 
 [IMPORTANT]
 ====
 Ensure that the first sentence explains the value and some benefits of the content in 26 words or less.
 ====
 
-If the first sentence is too long or can not be simplified to fit on the website tile, you can define a ``:summary:`` attribute in the document header attributes to serve that purpose. 
+If the first sentence is too long or can not be simplified to fit on the website tile, you can define a ``:summary:`` attribute in the document header attributes to serve that purpose.
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/doc-reference.adoc:197]:
```suggestion
If the first sentence is too long or cannot be simplified to fit on the website tile, you can define a ``:summary:`` attribute in the document header attributes to serve that purpose.
```

File: docs/src/main/asciidoc/doc-reference.adoc [hunk 2]
```diff
@@ -234,6 +243,10 @@ https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Sy
 The above source produces this link:https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Syntax Quick Reference,window=_blank,opts=nofollow].
 =====
 
+[[cross-references]]
+=== Cross-references
+
+
 [[cross-references]]
 === Cross-references
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/doc-reference.adoc:251]: Per Erin's comment above "Cross-references" is repeated. Perhaps this second instance can be removed.

File: docs/src/main/asciidoc/doc-reference.adoc [hunk 3]
```diff
@@ -303,11 +316,39 @@ xref:<other-file-name>.adoc#title-heading[Title heading]
...
+For more details about anchored IDs, see the xref:doc-contribute-docs-howto.adoc#anchors-howto[Cross-reference in-file and cross-file content by using anchors] section of the "Contribute to Quarkus documentation" guide.
+
+=== Cross-reference phrasing
+
+For consistency and context clarity, use the following guidelines for constructing your cross-reference phrases:
 
-=== Reference source code 
+* Provide a direct hyperlink to the section, using the `xref` guidance in the previous section.
+* Specify the full title of the section and use sentence case.
+* Add the definite article “the” before the section title and specify  “section” directly after the title.
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/doc-reference.adoc:327]:
```suggestion
* Add the definite article “the” before the section title and specify “section” directly after the title.
```

File: docs/src/main/asciidoc/doc-reference.adoc [hunk 4]
```diff
@@ -303,11 +316,39 @@ xref:<other-file-name>.adoc#title-heading[Title heading]
...
+* Add the definite article “the” and “Quarkus” before the title of the guide, unlessthe word “Quarkus" is already in the title.
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/doc-reference.adoc:328]:
```suggestion
* Add the definite article “the” and “Quarkus” before the title of the guide, unless the word “Quarkus" is already in the title.
```

File: docs/src/main/asciidoc/doc-reference.adoc [hunk 5]
```diff
@@ -234,6 +243,10 @@ https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Sy
 The above source produces this link:https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Syntax Quick Reference,window=_blank,opts=nofollow].
 =====
 
+[[cross-references]]
+=== Cross-references
+
```
[the remark this answers is not in the document]
Comment (michelle-purcell) [reply docs/src/main/asciidoc/doc-reference.adoc:263]: @ebullient - Thanks Erin for reviewing. Sorry I missed that :deciduous_tree: You are absolutely correct :+1: 

I pushed further change just now to promote the sections **Links**, **Cross-references**, and **Source code and configuration examples** from === to == and out of the Document structure section as they are big enough &  popular topics in their own right. WDYT? 

image

File: docs/src/main/asciidoc/doc-reference.adoc (hunk 2 above)
Comment (michelle-purcell) [reply docs/src/main/asciidoc/doc-reference.adoc]: Thanks, @Sheila. Oops. It is now removed :+1:

File: docs/src/main/asciidoc/doc-reference.adoc (hunk 4 above)
Comment (michelle-purcell) [reply docs/src/main/asciidoc/doc-reference.adoc]: @sheilamjones - Good catch! Thank you. I fixed this in my last commit with a batch of other changes. It's now appearing OK on line 344.

**Comment by michelle-purcell** [comment]
@sheilamjones & @ebullient - Thank you so much for your prompt reviews. I think I have resolved all of your comments.
If you are happy with the adjustment to the section headings then we can go ahead and merge this one.

**Review by rolfedh — APPROVED** [review]
LGTM! Great work, @michelle-purcell!

**Comment by ebullient** [comment]
There is a merge commit here that needs to be resolved.

**Review by ebullient — CHANGES_REQUESTED** [review]
Must resolve merge commit

**Comment by michelle-purcell** [comment]
@ebullient - Self-inflicted conflicts are now resolved :wink:

**Review by ebullient — APPROVED** [review]

**Pull request merged**